feat: support constant case (#557)
toss/es-toolkit: docs/zh_hans/reference/string/constantCase.md

# constantCase

Converts a string to constant case.

> Constant case is a naming convention where each word is written in uppercase letters and separated by an underscore (\_). Such as `CONSTANT_CASE`.

Reviewer: ```suggestion
Constant case is a naming convention where each word is written in uppercase letters and separated by an underscore (`_`). For example, `CONSTANT_CASE`.
```
